### DISCUSSION TITLE
I want side bar order change + question

### DISCUSSION THREAD

**Discussion by jongeuni** [imfing/hextra#279] [Q&A]
**Feature Description**
<img width="250" alt="image" src="https://github.com/imfing/hextra/assets/66874658/611f365c-ab6d-42a6-9f35-e245a37431ca">
I want to adjust the order of each folder.
Is there anything I'm missing? Please let me know if there's a way.

---

**Question**
I'd like to adjust the line spacing additionally.(enter) Please give me an idea through which file I can change it. 
I love this theme very much, please help me use it better.

**Comment by imfing** [comment]
@jongeuni Hi, you can set the `weight` parameter in the page frontmatter.

> I'd like to adjust the line spacing additionally.(enter) Please give me an idea through which file I can change it.

you can add custom css rules here: 

https://imfing.github.io/hextra/docs/advanced/customization/#custom-css

https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

for additional instructions, you could try searching on google or chatgpt 😃
